lib/CL/devices/level0/level0-driver.cc

```diff
@@ -2559,6 +2549,34 @@ bool Level0Device::setupModuleProperties(bool &SupportsInt64Atomics,
   SupportedSpvVersion =
       pocl_version_t(ZE_MAJOR_VERSION(SpvVer), ZE_MINOR_VERSION(SpvVer));
 
+  if (SpvVer == 0)
+    return true;
+
+  ClDev->compiler_available = CL_TRUE;
+  ClDev->linker_available = CL_TRUE;
+#ifdef ENABLE_GENERIC_AS
+  ClDev->generic_as_support = CL_TRUE;
+#endif
+
+#ifdef USE_LLVM_SPIRV_TARGET
+  LLVMTargetTriple = "spirv64v";
+  LLVMTargetTriple.push_back('0'+SupportedSpvVersion.major);
+  LLVMTargetTriple.push_back('.');
+  LLVMTargetTriple.push_back('0'+SupportedSpvVersion.minor);
+  LLVMTargetTriple.append("-unknown-unknown");
+#else
+  LLVMTargetTriple = "spir64-unknown-unknown";
+#endif
+  ClDev->llvm_target_triplet = LLVMTargetTriple.c_str();
+
+  for (int minor = SupportedSpvVersion.major; minor >= 0; --minor) {
```

Comment:
Did you mean to write:
```suggestion
  for (int minor = SupportedSpvVersion.minor; minor >= 0; --minor) {
```